src/request.rs

```diff
@@ -146,7 +138,7 @@ impl DispatchSignedRequest for Client {
 
         Ok(HttpResponse {
             status: hyper_response.status,
-            body: body,
+            body: Box::new(hyper_response),
```

Comment:
I'm hitting my limit of Rust knowledge on this: does `Box::new` copy the hyper response or create another pointer to it?  [The docs say it allocated space on the heap for the object](https://doc.rust-lang.org/std/boxed/struct.Box.html#method.new).  If it copies it, we'll have two copies of the object hanging around.  If it adds a pointer to the existing address on the heap it's fine.